WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -426,10 +426,18 @@ void updateNotificationSuccessForMedia(@NonNull List<MediaModel> mediaList, @Non
     }
 
     public static long getNotificationIdForPost(PostModel post) {
-        long remotePostId = post.getRemotePostId();
+        long postIdToUse = post.getRemotePostId();
+        if (post.isLocalDraft()) {
```

Comment:
I wonder if there could be the case where a local draft on the device, and a remote post, can get the same ID, since the ID is returned from 2 different sources.

Say, on self hosted blogs, where remote post ID start from 1 -  It's unlikely, but there could be a case where two different posts, both with media errors, can have the same ID. The first that comes from the local DB, and the second form the remote. Just wanted to point this out, not meaning to fix it in this PR.

Reply:
That's totally a fair concern, thanks for pointing that out so it's written somewhere - it's true that these IDs can in theory overlap, however the chances are too low so it seems it should work in practically 100% of times. Do you think maybe adding a comment in the code would be good, to alert future selves just in case?

Reply:
Yes, I think that just a short comment is OK.

Reply:
👌 addressed in 928a6501dd